hack/add/application.sh

```diff
@@ -18,11 +18,6 @@ repository:
 identities:
   - id: 1
   - id: 2
-facts:
-  - key: A
-    value: 1
-  - key: B
-    value: 2
```

Comment:
Why facts removed?

Reply:
The Application resource doesn't have a facts field, so these were just getting ignored. This PR turns on strict binding, so the unknown field causes a failure.

Reply:
ah, makes sense.